### ISSUE TITLE
Control of invitation links

### ISSUE THREAD

**Issue by MBudkin** [Ylianst/MeshCentral#5406]
You can add a client to a group of devices by downloading the Agent, or you can create an invitation link. But the problem is that the created links cannot be controlled after they are created (or I haven't found it).
That is, if I created a perpetual invitation link to download the Agent, then I will not be able to cancel it.
If I'm wrong, please correct me.

**Comment by si458** [comment]
This is correct, but I think I get where u are coming from
You want the ability to delete invite links incase say u create a forever link and only meant to create a 1 hour link?

**Comment by MBudkin** [comment]
@si458 
Yeah, something like that, thanks.

**Comment by si458** [comment]
ill keep it open as it is defo a feature request and a good one! i just need to look into it more

**Comment by si458** [comment]
> Is there an easy way to change the guide image in the invitiation link? 

Yes I can't remember which image it is, but simple create the folder `meahcentral-web` next to `meshcentral-data` then inside create `public` then copy the whole `images` folder from `node_modules/meshcentral/public/images` to `meshcentral-web/public/images`, edit ur images, then restart ur meshcentral server and the web ui will load ur custom images

**Comment by IceOmny** [comment]
> meahcentral-web

Thank you for the fast replay :)!
